sdk/eventhub/Azure.Messaging.EventHubs/samples/Program.cs

```diff
@@ -74,30 +229,119 @@ public static async Task Main(string[] args)
                 Console.WriteLine($"\t{ samples[index].Description }");
                 Console.WriteLine();
             }
+        }
 
-            Console.WriteLine();
-            var choice = ReadSelection(samples.Count);
+        /// <summary>
+        ///   Prompt the user to insert the EventHubs connection string, if not
+        ///   already passed it from command line.
+        /// </summary>
+        ///
+        /// <param name="parsedArgs">The arguments passed from console.</param>
+        ///
+        private static void PromptConnectionString(CommandLineArguments parsedArgs)
+        {
+            // Prompt for the connection string, if it wasn't passed.
 
-            if (choice == null)
+            while (String.IsNullOrEmpty(parsedArgs.ConnectionString))
             {
+                Console.Write("Please provide the connection string for the Event Hubs namespace that you'd like to use and then press Enter: ");
+                parsedArgs.ConnectionString = Console.ReadLine().Trim();
                 Console.WriteLine();
-                Console.WriteLine();
-                Console.WriteLine("Quitting...");
-                Console.WriteLine();
-                return;
             }
-            else
+        }
+
+        /// <summary>
+        ///   Prompt the user to insert the EventHubs fully qualified name, if not
+        ///   already passed it from command line.
+        /// </summary>
+        ///
+        /// <param name="parsedArgs">The arguments passed from console.</param>
+        ///
+        private static void PromptEventHubName(CommandLineArguments parsedArgs)
```

Comment:
I love this approach!   One small nit, since it does a conditional check before prompting, it likely makes sense to revise to `PromptIfMissing` or similar?